### PR TITLE
Add CLI deps to polymer.json

### PR DESCRIPTION
**Pull request by robdodson**
See https://github.com/Polymer/polymer-cli/issues/386

@FredKSchott PTAL

### CODE REVIEW

**Comment by FredKSchott** [comment]
LGTM! Good catch

File: polymer.json [hunk 1]
```diff
@@ -7,5 +7,14 @@
...
     "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "includeDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
```
Comment (frankiefu) [inline polymer.json]: @robdodson Thanks for the fix!

@robdodson and @FredKSchott, we also need to copy the `data` folder to the build.  Should we put that under `sourceGlobs`?

Comment (frankiefu) [inline polymer.json]: Btw, what's the difference between `sourceGlobs` and `includeDependencies`?

Comment (FredKSchott) [inline polymer.json]: Great question! We're actually still figuring that out as we go. If you're using the 'polymer-build' library yourself you can apply different optimizations/transforms to source files and dependency files. But today in the CLI we apply the same optimizations to both, so there's no real difference if you're using 'polymer build' on the command line.

We've seen this question a few times now, we should spend some time clarifying this in the README.

Comment (FredKSchott) [inline polymer.json]: As for 'data/' yes, if you need it in the build and it's not picked up by the analyzer via HTML imports/loading then you should include it here

Comment (frankiefu) [inline polymer.json]: @FredKSchott Thanks man!
@robdodson Do you mind updating to add `"data/**/*"` in the PR?

**Comment by robdodson** [comment]
yep added

**Comment by frankiefu** [comment]
Thanks, LGTM.

**Pull request merged**